.github/workflows/hugo.yml

```diff
@@ -69,6 +73,7 @@ jobs:
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     needs: build
+    if: github.ref == 'refs/heads/main'
```

Comment:
Should this be `v2`? I don't think we have a main branch.

Reply:
Yes! This would have stopped deployment. I am going to double check but I think `HEAD` is an alias for the default branch, which would make this less fragile (or safer since if we change the default branch, we might not expect that to change how deployment works)

Reply:
> or safer since if we change the default branch, we might not expect that to change how deployment works

seems like a bigger footgun, will switch to just using the right branch name